gulp-core-build/src/IExecutable.ts

```diff
@@ -8,7 +8,7 @@ export interface IExecutable {
   name?: string;
 
   /** Optional callback to indicate if the task is enabled or not. */
```

Comment:
For props like this which takes in callbacks, good to document what kind of callback signature is expected..